### PR TITLE
Created test case

### PR DESCRIPTION
**Pull request by Svyeta0210**
Created test case on registrationNavigation() using @Test annotation

### CODE REVIEW

**Comment by Berichicko** [comment]
Please, add Homework16 in TestNG.xml to run tests

image

File: src/test/java/Homework16.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+     //Assert Registration page is Displayed
+     WebElement registrationPage = driver.findElement(By.xpath(("//div[@class='login-wrapper']")));
+     Assert.assertTrue(registrationPage.isDisplayed());
+     //close the browser
+     driver.quit();
+
+ }
+
+
+}
```
Comment (Berichicko) [inline src/test/java/Homework16.java:34]: Please finish the homework!You got this problem, please correct it. 

image

Comment (Svyeta0210) [reply src/test/java/Homework16.java:34]: Hello Alex,
I do not understand what is the problem with the file name. Should I have created homework16 file in class/file or just 'file' in intelliJ? Is this the reason why the code doesn't run? 
Thank you.

**Comment by Berichicko** [comment]
Hello 
Your homework 16 is checked
**You’re doing great! Keep it up! Very fine!** 
💖Best wishes, the team HW (Alex,  aliaksandr@testpro.io)!💖

giphy


image

**Pull request closed without merging**